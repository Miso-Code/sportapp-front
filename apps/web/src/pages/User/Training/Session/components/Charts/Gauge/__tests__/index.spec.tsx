import GaugeChart from '..'
import { render, RenderResult } from '@testing-library/react'

jest.mock('@mui/x-charts/PieChart', () => ({
	PieChart: () => <div>PieChart</div>
}))

describe('GaugeChart', () => {
	let wrapper: RenderResult

	beforeEach(() => {
		wrapper = render(<GaugeChart maxValue={100} value={50} />)
	})

	it('should render the component', () => {
		expect(wrapper.container).toMatchSnapshot()
	})
})
