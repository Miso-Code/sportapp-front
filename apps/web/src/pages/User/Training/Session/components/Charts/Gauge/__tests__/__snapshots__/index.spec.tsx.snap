// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GaugeChart should render the component 1`] = `
<div>
  <div
    class="MuiBox-root css-0"
    style="position: relative; width: 200px; height: 200px;"
  >
    <div>
      PieChart
    </div>
    <div
      class="MuiBox-root css-adxcwc"
    />
  </div>
</div>
`;
